core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/builtin_argument/argument_values_set.go

```diff
@@ -135,6 +135,13 @@ func parseArguments(argumentDefinitions []*BuiltinArgument, builtinName string,
 		if argumentDefinition.IsOptional && argValue == nil {
 			continue
 		}
+		if argumentDefinition.Validator != nil {
```

Comment:
why do we move this around?

Reply:
because for the wait attribute, we receive an interface type `starlark.Value` and it was caught on this part of the code `if reflect.TypeOf(argumentDefinition.ZeroValueProvider()) == nil {
			// it means the type is an interface, we are not able to validate its concrete type at this step
			continue
		}`. so I moved this piece of code up in order to execute the validator without being caught by the mentioned condition before.